ntp-udp/src/socket.rs

```diff
@@ -179,7 +179,7 @@ impl UdpSocket {
                     .await
                 {
                     Err(_) => {
-                        warn!("Packet without timestamp");
+                        tracing::info!("Packet without timestamp");
                         Ok((send_size, None))
```

Comment:
This feels to me like it should be a warning, as we are not getting from the operating system what we asked for, and the user likely does want to resolve this.